src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Participants/index.tsx

```diff
@@ -1,26 +1,344 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import { Controller } from 'react-hook-form';
+import { Trans, useTranslation } from 'react-i18next';
+import { useHistory, useLocation } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  Button,
+  ButtonGroup,
+  Fieldset,
+  FormGroup,
+  Grid,
+  ModalFooter,
+  ModalHeading
+} from '@trussworks/react-uswds';
+import {
+  GetSystemIntakeGRBReviewDocument,
+  SystemIntakeGRBReviewerFragment,
+  SystemIntakeGRBReviewFragment,
+  useDeleteSystemIntakeGRBReviewerMutation,
+  useUpdateSystemIntakeGRBReviewFormInputTimeframeAsyncMutation
+} from 'gql/generated/graphql';
 
+import DatePickerFormatted from 'components/DatePickerFormatted';
 import { EasiFormProvider, useEasiForm } from 'components/EasiForm';
+import FieldErrorMsg from 'components/FieldErrorMsg';
+import Modal from 'components/Modal';
+import RequiredAsterisk from 'components/RequiredAsterisk';
+import useMessage from 'hooks/useMessage';
 import { GRBReviewFormStepProps } from 'types/grbReview';
+import { SetGRBParticipantsAsyncSchema } from 'validations/grbReviewSchema';
 
+import ParticipantsTable from '../../ParticipantsSection/_components/ParticipantsTable';
 import GRBReviewFormStepWrapper, {
   GRBReviewFormStepSubmit
 } from '../GRBReviewFormStepWrapper';
 
-// TODO: Update fields type
-type ParticipantsFields = {};
+type ParticipantsFields = {
+  grbReviewers: SystemIntakeGRBReviewerFragment[];
+  grbReviewAsyncEndDate: SystemIntakeGRBReviewFragment['grbReviewAsyncEndDate'];
+};
 
 const Participants = ({ grbReview }: GRBReviewFormStepProps) => {
-  const form = useEasiForm<ParticipantsFields>();
+  const { t } = useTranslation('grbReview');
+
+  const [mutate] =
+    useUpdateSystemIntakeGRBReviewFormInputTimeframeAsyncMutation();
+
+  const form = useEasiForm<ParticipantsFields>({
+    resolver: yupResolver(SetGRBParticipantsAsyncSchema),
+    defaultValues: {
+      grbReviewers: grbReview.grbReviewers,
+      grbReviewAsyncEndDate: grbReview.grbReviewAsyncEndDate || ''
+    },
+    values: {
+      grbReviewers: grbReview.grbReviewers,
+      grbReviewAsyncEndDate: grbReview.grbReviewAsyncEndDate || ''
+    }
+  });
+  const {
+    control,
+    formState: { errors }
+  } = form;
+
+  const onSubmit: GRBReviewFormStepSubmit<ParticipantsFields> = async input => {
+    return mutate({
+      variables: {
+        input: {
+          systemIntakeID: grbReview.id,
+          grbReviewAsyncEndDate: input.grbReviewAsyncEndDate ?? ''
+        }
+      }
+    });
+  };
+
+  const history = useHistory();
+  const { pathname } = useLocation();
+  const { showMessage } = useMessage();
```

Comment:
Just a style nitpic that these hooks are normally defined first in the file, rather than midway through